docs/how-to-docs/keycloak-sidecar.md

```diff
@@ -0,0 +1,109 @@
+# Using Keycloak Sidecar
+
+The keycloak gatekeeper (also know as the keycloak sidecar) is used to enable SSO on applications. [More information on configuration options](https://www.keycloak.org/docs/8.0/securing_apps/#configuration-options)
```

Comment:
It's not just for SSO. It's a general purpose adapter for Keycloak. You could also mention its future name - https://github.com/louketo/louketo-proxy

Reply:
made the change